lib/webpacker/dev_server_runner.rb

```diff
@@ -14,7 +15,7 @@ def run
     private
       def load_config
         @config_file = File.join(@app_path, "config/webpacker.yml")
-        dev_server = YAML.load_file(@config_file)[ENV["RAILS_ENV"]]["dev_server"]
+        dev_server = YAML.load(ERB.new(File.read(@config_file)).result)[ENV["RAILS_ENV"]]["dev_server"]
```

Comment:
Not sure how to test this change (except from checking manually).

I think, it make sense to re-use `Configuration` here (but that would require its refactoring to not depend on `Webpacker` instance).